x-pack/plugins/apm/server/lib/helpers/create_es_client/create_apm_event_client/unpack_processor_events.test.ts

```diff
@@ -19,6 +19,12 @@ describe('unpackProcessorEvents', () => {
     } as APMEventESSearchRequest;
 
     const indices = {
+      'xpack.apm.transactionIndices': 'my-apm-*-transaction-*',
+      'xpack.apm.metricsIndices': 'my-apm-*-metric-*',
+      'xpack.apm.errorIndices': 'my-apm-*-error-*',
+      'xpack.apm.spanIndices': 'my-apm-*-span-*',
+      'xpack.apm.onboardingIndices': 'my-apm-*-onboarding-',
+      'xpack.apm.sourcemapIndices': 'my-apm-*-sourcemap-*',
       'apm_oss.transactionIndices': 'my-apm-*-transaction-*',
       'apm_oss.metricsIndices': 'my-apm-*-metric-*',
```

Comment:
Should the `apm_oss` settings still be there?

Reply:
i can remove them from the tests.

Reply:
done